.gitmodules

```diff
@@ -15,27 +15,28 @@
 [submodule "kubelet"]
 	path = kubelet
 	url = https://github.com/openshift/kubernetes
-	branch = release-4.16
+	branch = release-4.17
 [submodule "kube-proxy"]
 	path = kube-proxy
 	url = https://github.com/openshift/kubernetes
 	branch = sdn-4.16-kubernetes-1.29.0
```

Comment:
Could you remind me when this is bumped/when we engage SDN to create the branch?

Reply:
It needs to be created after the kube rebase

Reply:
The proposal was for the WINC team to maintain them :)